packages/perseus/src/widgets/__tests__/radio/choice_test.jsx

```diff
@@ -30,66 +30,105 @@ function renderChoice(options) {
     return render(<Choice {...overwrittenOptions} />);
 }
 
-describe("choice", () => {
-    it("renders choice content", () => {
+describe("all choice options", () => {
+    it("renders rationale", () => {
```

Comment:
It took me a second to realize that some of these changes were due to re-ordering of tests.  Is there a reason you re-ordered these?  I'm not opposed to it, but usually those types of things should be done in their own PR so that the important changes in this PR aren't "muddied" by larger changes from re-organizing things.

Reply:
Thanks for pointing this out! I kept thinking I should add a comment, but I then kept forgetting. It was because I added an additional section for tests specific to the native radio options. I think we chatted about this a little actually. I wanted to create categories for the tests, so I ended up creating a category for all choices, for choice buttons, and another for choice radios. Since I needed to add the radio tests, I wanted it to be clear which tests were for which element. Unfortunately, I did not realize it would look like this in the PR.